Makefile

```diff
@@ -0,0 +1,33 @@
+DEFAULT_VERSION = ''
```

Comment:
what will be the strategy to run this test?

till now we had package.json and we used https://github.com/asyncapi/github-action-for-generator/blob/master/.github/workflows/if-nodejs-pr-testing.yml

maybe best would be to leave package.json, even if just there for running tests 🤔 to not have to much CI/CD maintenance on our side. So basically a script in package.json that triggers the make file when `npm run test` is triggered?

wdyt, then we can still use global file https://github.com/asyncapi/github-action-for-generator/blob/master/.github/workflows/if-nodejs-pr-testing.yml

Reply:
A quick question:- Do we need js tests? It doesn't make sense when we are using the shell to run everything.

Reply:
we definitely do not need to have js tests

I'm suggesting `package.json` not to have js tests but to be able to reuse global test action that depends on `package.json` - you can still have other tests, make files - just trigger of test will go through package.json

so if you keep package.json and have a `test` script there, we can still use https://github.com/asyncapi/github-action-for-generator/blob/master/.github/workflows/if-nodejs-pr-testing.yml as it triggers test script in https://github.com/asyncapi/github-action-for-generator/blob/master/.github/workflows/if-nodejs-pr-testing.yml#L72

does that make sense?

or maybe we do not need MAKE file, and test should be basically a workflow that represents all possible options and runs on PR

have a look at https://dev.to/cicirello/how-to-test-a-github-action-with-github-actions-2hag 

it works this way that in PR workflow, you do instead of `uses: docker://asyncapi/github-action-for-generator:2.0.0` something like `uses: ./` so like run a workflow with steps that use action version that is in the PR - you know what I mean?